frontend/src/components/Admin/DevPortfolio/DevPortfolioDetails.tsx

```diff
@@ -89,6 +89,9 @@ const DevPortfolioDetails: React.FC<Props> = ({ uuid, isAdminView }) => {
     csvExporter.generateCsv(csvData);
   };
 
+  // console.log(new Date(portfolio?.lateDeadline));
```

Comment:
Let's remove this commented out line here.

Reply:
👍